vnext/Folly/Folly.vcxproj

```diff
@@ -67,6 +67,35 @@
     <ClCompile Include="$(FollyDir)\folly\Unicode.cpp" />
     <ClCompile Include="$(FollyDir)\folly\memory\detail\MallocImpl.cpp" />
   </ItemGroup>
+  <ItemGroup Condition="'$(EnableHermesInspector)' == 'true'">
+    <ClCompile Include="$(FollyDir)\folly\ExceptionWrapper.cpp" DisableSpecificWarnings="4541;4459;%(DisableSpecificWarnings)" />
+    <ClCompile Include="$(FollyDir)\folly\Executor.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\SharedMutex.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\concurrency/CacheLocality.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\detail/AsyncTrace.cpp" DisableSpecificWarnings="4541;4459;%(DisableSpecificWarnings)"/>
+    <ClCompile Include="$(FollyDir)\folly\detail/Futex.cpp" DisableSpecificWarnings="4541;4459;%(DisableSpecificWarnings)"/>
+    <ClCompile Include="$(FollyDir)\folly\detail/MemoryIdler.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\detail/StaticSingletonManager.cpp" DisableSpecificWarnings="4541;4459;%(DisableSpecificWarnings)" />
+    <ClCompile Include="$(FollyDir)\folly\detail/ThreadLocalDetail.cpp" />
+    <ClCompile Include="$(FollyDir)\folly/detail/AtFork.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\detail\UniqueInstance.cpp" DisableSpecificWarnings="4541;4459;%(DisableSpecificWarnings)"/>
+    <ClCompile Include="$(FollyDir)\folly\executors/ExecutorWithPriority.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\executors/InlineExecutor.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\executors/TimedDrivableExecutor.cpp" DisableSpecificWarnings="4541;4459;%(DisableSpecificWarnings)"/>
```

Comment:
4541 results in undefined behavior due to needing RTTI
4459 is local declaration hides outer scope's
Are these fixed in newer folly versions?

Reply:
Some of these are still present in newer folly version .. Please take a look at the new PR .. We are now not downgrading the headers ..